USER-GUIDE.md

```diff
@@ -0,0 +1,484 @@
+# KCP-Edge User Guide:
+
+## Required Packages:
+   - docker
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Setup Instructions
+
+This guide is intended to explain how KCP-Edge works with a detailed example.
+
+### 1. Create your own edge infrastructure (edge clusters) - Bring Your Own Clusters (BYOC)
+
+Bring your own k8s edge clusters. In this example, we deploy only a single cluster using [kind](https://kind.sigs.k8s.io/). Our edge cluster is named “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 2. Install KCP
+
+Since `KCP-Edge` leverages [`kcp`](kcp.io) logical workspaces capability, we first need to install `kcp` v0.11.0 binaries. To install kcp follow the instructions available [on the kcp website](https://docs.kcp.io/kcp/main/#download-kcp). You can also use the script below:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-with-plugins.sh) --version v0.11.0 --folder $(pwd)/kcp --create-folder
```

Comment:
If this were instead

```
curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-with-plugins.sh | source /dev/stdin --version v0.11.0 --folder $(pwd)/kcp --create-folder
```

then the script could set PATH too.

Reply:
@francostellari, could you look at the script to address this comment? thanks